code/modules/mob/living/carbon/carbon.dm

```diff
@@ -772,6 +775,10 @@
 	update_health_hud()
 	med_hud_set_status()
 
+mob/living/carbon/proc/do_fortnite()	//The assistant is dying in maintenance with no sensors! I need the number on the front of your moms credit card, the three digit number on the back and the expiry date to save them.
+	if(stat == CONSCIOUS) //*floss

```

Comment:
when would this ever be called when you're conscious?

Reply:
This is necessary, updated with comment as to why